squad/ci/backend/null.py

```diff
@@ -92,18 +92,24 @@ def cancel(self, test_job):
         """
         raise NotImplementedError
 
-    def job_url(selt, test_job):
+    def job_url(self, test_job):
         """
         Returns the URL of the test job in the backend
         """
         raise NotImplementedError
 
-    def check_job_definition(selt, definition):
+    def check_job_definition(self, definition):
         """
         Returns True if job definition checks out or an error message
         """
         raise NotImplementedError
 
+    def get_job_definition(self, test_job):
```

Comment:
Can you make this `job_id` instead of `test_job` as it only uses the job id?

Reply:
Fixed with [f28d3b7](https://github.com/Linaro/squad/pull/1123/commits/f28d3b75d0838a956d46611ba79c1dc640ff18ce)